packages/react-reconciler/src/__tests__/ReactFiberCommit-test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+'use strict';
+
+let React;
+let ReactFiberReconciler;
+
+describe('ReactFiberHostContext', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    React = require('react');
+    ReactFiberReconciler = require('react-reconciler');
+  });
+
+  it('should take the data returned by prepareForCommit and send them back to resetAfterCommit', () => {
+    let savedCommitData;
+    const Renderer = ReactFiberReconciler({
+      prepareForCommit: function(hostContext) {
+        return (savedCommitData = {});
+      },
+      resetAfterCommit: function(hostContext, commitData) {
+        expect(commitData).toBe(savedCommitData);
```

Comment:
Could you try to come up with a test case that's more representative of a real world use case? As suggested in [Sebastian's comment](https://github.com/facebook/react/pull/12098#issuecomment-360688768). Ideally a person should be able to look at this unit test and understand not just what the method does, but why it exists.

Reply:
>  it will be handy when I will get to implementing selection inside the renderer

This sounds interesting. If you can approximate this use case in the unit test, maybe with some comments to clarify, I'll merge the PR :)

Reply:
Since I don't use it in my renderer at the moment I removed the commit data from this PR, I'll open a new one when I'll reach the point where it is needed :)